pac_darwin.go

```diff
@@ -0,0 +1,38 @@
+package pac
+
+/*
+#cgo darwin CFLAGS: -DDARWIN -x objective-c
+#cgo darwin LDFLAGS: -framework Cocoa -framework Security
+
+extern int runAuthorized(char *helperFullPath, char *prompt, char *iconFullPath);
+const char* null() { return 0; }
+*/
+import "C"
+import (
+	"fmt"
+	"syscall"
+)
+
+func ensureElevatedOnDarwin(helperFullPath string, prompt string, iconFullPath string) (err error) {
+	var s syscall.Stat_t
```

Comment:
This works.  FYI, you could have also used os.Stat and looked at the FileMode on the returned FileInfo.  There is no need to change this though, just an FYI.